src/gt4py/next/iterator/ir_utils/ir_makers.py

```diff
@@ -239,15 +241,42 @@ def tuple_get(index: str | int, tuple_expr):
 
 
 def if_(cond, true_val, false_val):
-    """Create a not_ FunCall, shorthand for ``call("if_")(expr)``."""
+    """Create a if_ FunCall, shorthand for ``call("if_")(expr)``."""
     return call("if_")(cond, true_val, false_val)
 
 
+def cond(cond, true_val, false_val):
+    """Create a cond FunCall, shorthand for ``call("cond")(expr)``."""
+    return call("cond")(cond, true_val, false_val)
+
+
 def lift(expr):
     """Create a lift FunCall, shorthand for ``call(call("lift")(expr))``."""
     return call(call("lift")(expr))
 
 
+def as_fieldop(expr: itir.Expr, domain: Optional[itir.FunCall] = None) -> call:
```

Comment:
Already in main: https://github.com/GridTools/gt4py/blob/main/src/gt4py/next/iterator/ir_utils/ir_makers.py#L438